examples/mobile/HomeApp/webclip/video-service/webclip.html

```diff
@@ -0,0 +1,60 @@
+<div class="ui-card ui-card-service" class="app-video-control">
+	<link rel="stylesheet" href="css/style.css" />
+	<div class="ui-content-subheader">
+		Video Collection
+	</div>
+	<div class="ui-content-area">
+		<div class="ui-section-changer">
+			<div class="ui-section-changer-container">
+				<section>
+					<div class="ui-content">
+						<video muted autoplay src="https://cdn.videvo.net/videvo_files/video/free/2012-07/small_watermarked/hd0936_preview.webm"
+							style="width: 100%" onclick="this.play()"></video>
+						<div class="ui-title">Ch2 : Aqua World</div>
+						<div class="ui-description">Established in the early 21st Century,
+							Aqua World is a brand synonymous with aqua tourism.</div>
+					</div>
+					<div class="ui-footer">
+						<button class="ui-btn-contained-colored" data-inline="true" onclick="playOnTV(1,0)">Play On TV</button>
+					</div>
+				</section>
+
+				<section class="ui-content-area">
+					<div class="ui-content">
+						<video muted autoplay src="https://videos.files.wordpress.com/4CjZLK5X/sample-video-1_hd.mp4"
+							style="width: 100%" onclick="this.play()"></video>
+						<div class="ui-title">Ch3 : Adventure Movie</div>
+						<div class="ui-description"> Action & Adventure. These movies all make for one amped-up viewing experience</div>
+					</div>
+					<div class="ui-footer">
+						<button class="ui-btn-contained-colored" data-inline="true" onclick="playOnTV(2,0)">Play On TV</button>
+					</div>
+				</section>
+
+				<section class="ui-content-area">
+					<div class="ui-content">
+						<video muted autoplay src="https://file-examples-com.github.io/uploads/2017/04/file_example_MP4_480_1_5MG.mp4" loop
+							id="video4" style="width: 100%" onclick="this.play()"></video>
+						<div class="ui-title">Ch4 : Toy World</div>
+						<div class="ui-description">Funny video for children to enjoy! Subscribe for notifications to new videos</div>
+					</div>
+					<div class="ui-footer">
+						<button class="ui-btn-contained-colored" data-inline="true" onclick="playOnTV(3,0)">Play On TV</button>
+					</div>
+				</section>
+
+				<section class="ui-content-area">
+					<div class="ui-content">
+						<video muted autoplay src="https://file-examples-com.github.io/uploads/2017/04/file_example_MP4_480_1_5MG.mp4" loop
+							id="video4" style="width: 100%" onclick="this.play()"></video>
```

Comment:
It's same video with "Ch4 : Toy World". Would you change the video like "http://clips.vorwaerts-gmbh.de/VfE_html5.mp4" , and title is "Ch1: BigBuckBunny" ,please?

Reply:
I'll change.

There will be potential issue with "http" protocol if home app will be open with "https". This source is available only by "http".

Reply:
Done.